src/file.ts

```diff
@@ -15,22 +15,25 @@ export const COMMON_MIME_TYPES = new Map([
 ]);
 
 
-export function toFileWithPath(file: File, path?: string): FileWithPath {
+export function ensureFileWithPath(file: FileWithPath, path?: string): FileWithPath {
```

Comment:
I don't see the point in renaming this fn 😕 It just adds unnecessary changes. Please undo,